wherescape/connectors/gitlab/readme.md

```diff
@@ -0,0 +1,54 @@
+# Gitlab connector
+
+Gitlab connector for WhereScape. Takes care of reading projects, issues, tags,
+pipelines and merge requests from a Gitlab instance.
+
+# Preparation
+
+## WhereScape Parameters
+
+Add the following parameters to WhereScape:
+* `gitlab_base_url`
+* `gitlab_access_token`
+* `gitlab_high_water_mark`
+* `gitlab_high_water_mark_next`
```

Comment:
These do not conform to the standard naming for HWMs . also, the set function should be called from the data-store table